packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -368,7 +368,42 @@ export const useArbTokenBridge = (
         decimals,
         logoURI
       } = tokenData
-      const l1Address = (extensions as any).l1Address as string
+
+      const l1Addresses = (() => {
+        // TODO: parsing the token list format could be from arbts or the tokenlist package
+        interface Extensions {
```

Comment:
we have this ts interface over in the token list package, we can import from there